lib/tutorials/en_US/caching.md

```diff
@@ -70,13 +87,19 @@ return h.response(result).etag('xxxxxxxxx');
 
 Check the documentation of `etag` under the [response object](/api#response-object) for more details about the arguments and available options.
 
-### Server side caching
+## <a name="server-side" /> Server-side Caching
 
-hapi provides powerful, convenient server side caching via [catbox](https://www.github.com/hapijs/catbox). This tutorial section will help you understand how to use catbox.
+hapi provides powerful, convenient server side caching via [catbox](https://www.github.com/hapijs/catbox). This tutorial section will help you understand how to use `catbox`.
```

Comment:
Any reason "catbox" is displayed as `code` one place and as text in another?  I don't mind one or the other, but I think consistency is nice 👍